raster/src/main/scala/geotrellis/raster/rasterize/polygon/OldPolygonRaserizer.scala

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright (c) 2014 Azavea.
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package geotrellis.raster.rasterize.polygon
+
+import geotrellis.raster._
+import geotrellis.vector._
+import geotrellis.raster.rasterize._
+
+object OldPolygonRasterizer {
```

Comment:
Before we merge the PR, we have to get rid of this. Feel free to move it into the `geotrellis-benchmark` project, so that we can still benchmark against it.